test/server/cards/14.4-CoC/StoicRival.spec.js

```diff
@@ -0,0 +1,85 @@
+                this.shibaTsukune = this.player2.findCardByName('shiba-tsukune');
+                this.miyaMystic = this.player2.findCardByName('miya-mystic');
+                this.stoicRival = this.player1.findCardByName('stoic-rival');
+                this.player2.playAttachment('ornate-fan','shiba-tsukune');
+                this.player1.playAttachment('vine-tattoo','stoic-rival');
+            });
+
+
+            it('should allow bowing cards with fewer attachments', function() {
+                this.player2.pass();
+                this.stoicRival = this.player1.clickCard(this.stoicRival);
+                expect(this.player1).toHavePrompt('Stoic Rival');
+                expect(this.player1).toBeAbleToSelect(this.otomoCourtier);
+            });
+
+            it('no bowing cards with same number of attachments', function() {
+                this.player2.pass();
+                this.stoicRival = this.player1.clickCard(this.stoicRival);
+                expect(this.player1).toHavePrompt('Stoic Rival');
+                expect(this.player1).not.toBeAbleToSelect(this.shibaTsukune);
+            });
+
+            it('no bowing cards with more attachments', function() {
+                this.player2.playAttachment('ornate-fan','shiba-tsukune');
+                this.stoicRival = this.player1.clickCard(this.stoicRival);
+                expect(this.player1).toHavePrompt('Stoic Rival');
+                expect(this.player1).not.toBeAbleToSelect(this.shibaTsukune);
+            });
+
+            it('should allow allow bowing cards with an attachment but still fewer', function() {
+                this.player2.pass();
+                this.player1.playAttachment('ornate-fan', this.stoicRival);
+                this.player2.pass();
+                this.player1.clickCard(this.stoicRival);
+                expect(this.player1).toHavePrompt('Stoic Rival');
+                expect(this.player1).toBeAbleToSelect(this.shibaTsukune);
+            });
+
+            it('should dishonor the target', function() {
+                this.player2.pass();
+                this.player1.clickCard(this.stoicRival);
+                this.player1.clickCard(this.otomoCourtier);
+                expect(this.otomoCourtier.isDishonored).toBe(true);
+                expect(this.otomoCourtier.inConflict).toBe(true);
+            });
+
+            it('should not be able to target characters not participating', function() {
+                this.player2.pass();
+                this.player1.clickCard(this.stoicRival);
+                expect(this.player1).toHavePrompt('Stoic Rival');
+                expect(this.player1).not.toBeAbleToSelect(this.miyaMystic);
+            });
+
+            it('should not be able to use while not participating', function() {
+                this.player2.clickCard('outwit');
+                this.player2.clickCard(this.stoicRival);
+                expect(this.stoicRival.inConflict).toBe(false);
```

Comment:
You can't outwit using the Otomo Courtier.
Stoic Rival (1 pol base) + Vine Tattoo (1 pol) puts him equal with Otomo Courtier.  Outwit needs < pol.

Reply:
I was wondering why the hell it wasn't working! I wanted attachments that were as blank and irrelevant as possible initially and forgot about it! Thanks!